test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8sClient.java

```diff
@@ -193,13 +193,13 @@ public CompletableFuture<V1PodStatus> getStatusOfPod(final String namespace, fin
         CoreV1Api api = new CoreV1Api();
         K8AsyncCallback<V1PodList> callback = new K8AsyncCallback<>("listPods");
         api.listNamespacedPodAsync(namespace, PRETTY_PRINT, ALLOW_WATCH_BOOKMARKS, null, null, "POD_NAME=" + podName, null,
-                                   null, null, false, callback);
+                null, null, DEFAULT_TIMEOUT_MINUTES * 60, false, callback);
```

Comment:
This is the timeout for watch/list the pods. Now we are making it default as we were not returning back from the watching state and the dependent test was failing because the test timeout was happening before watch return. the related test is `ReadTxnWriteScaleWithFailoverTest` and the exception looks like this 
```Watch did not return terminated state for pod readtxnwritescalewithfailovertest-zopwe```